src/crypto/qat/qcccrypto.h

```diff
@@ -6,9 +6,21 @@
 #include <stdlib.h>
 #include <unistd.h>
 #include <pthread.h>
+#include <thread>
+#include <mutex>
 #include <queue>
+#include <memory>
+#include "common/async/yield_context.h"
+#include "common/async/completion.h"
+#include "include/rados/librados_fwd.hpp"
```

Comment:
what does this need librados for?

Reply:
> what does this need librados for?

Here don't need `librados_fwd.hpp`. I have removed it.